.eslintrc.yml

```diff
@@ -0,0 +1,30 @@
+env:
+  browser: true
+  es2021: true
+  jest: true
+extends:
+  - eslint:recommended
+  - plugin:react/recommended
+  - plugin:@typescript-eslint/recommended
+  - prettier
```

Comment:
Don't you want to take one of the ready-to-go code-style configs such as `airbnb` and `airbnb-typescript`?